fdbserver/storageserver.actor.cpp

```diff
@@ -179,13 +179,14 @@ struct StorageServerDisk {
 	StorageBytes getStorageBytes() const { return storage->getStorageBytes(); }
 	std::tuple<size_t, size_t, size_t> getSize() const { return storage->getSize(); }
 	
-	bool canPipelineCommits() const {return storage->canPipelineCommits();}
+	bool canPipelineCommits() const {return _canPipelineCommits;}
 	void set(KeyValueRef kv) { storage->set(kv);}
 	void clear(KeyRangeRef kr) { storage->clear(kr);}
 
 private:
 	struct StorageServer* data;
 	IKeyValueStore* storage;
+	bool _canPipelineCommits;
```

Comment:
This could be declared const, since it is initialized in the constructor initializer list and is not allowed to change.